server/service/src/sync/translations/name.rs

```diff
@@ -127,27 +136,42 @@ pub struct LegacyNameRow {
 
     #[serde(rename = "isDeceased")]
     pub is_deceased: bool,
+
     #[serde(rename = "om_created_datetime")]
     #[serde(deserialize_with = "empty_str_as_option")]
     pub created_datetime: Option<NaiveDateTime>,
+
     #[serde(rename = "om_gender")]
     #[serde(deserialize_with = "empty_str_as_option")]
     pub gender: Option<GenderType>,
-    #[serde(rename = "om_date_of_death")]
+
     #[serde(default)]
+    #[serde(rename = "om_date_of_death")]
     #[serde(deserialize_with = "zero_date_as_option")]
     #[serde(serialize_with = "date_option_to_isostring")]
     pub date_of_death: Option<NaiveDate>,
+
+    #[serde(default)]
     pub custom_data: Option<serde_json::Value>,
+
+    #[serde(default)]
```

Comment:
Added default to all these fields incase they're missing...

Reply:
If it's optional doesn't it get deserialised as `None` if it's missing anyway? How is default different?

Reply:
At least the HSH_CODE one breaks. Maybe it's if we're using a custom translator? I'll leave it how it is for now.
```
thread 'sync::translations::name::tests::test_name_translation' panicked at service/src/sync/translations/name.rs:469:18:
Error translating from upsert sync record "ef232e6c-93cf-4d02-a95a-c563abccc24f": missing field `HSH_code` at line 33 column 1
```